src/sdk/model/edge/certificates/certificate-create-request.ts

```diff
@@ -0,0 +1,51 @@
+import { CertificateCreateRequestJson } from './__json__/certificate-create-request-json';
+import { TrustObjectRequestJson } from './__json__/trust-object-request-json';
+import { TrustObjectRequest } from './trust-object-request';
+
+/* istanbul ignore next: autogenerated */
+export class CertificateCreateRequest {
+
+  private readonly _json: CertificateCreateRequestJson;
+
+  constructor(certificateCreateRequest: CertificateCreateRequest);
+  constructor(certificateCreateRequestJson: CertificateCreateRequestJson);
+  constructor(trustObject: TrustObjectRequest);
+  constructor(firstParam: TrustObjectRequest | CertificateCreateRequest | CertificateCreateRequestJson) {
+    if (firstParam instanceof TrustObjectRequest) {
+      // Parameters constructor
+      this._json = {
+        trust_object: firstParam.json
+      } as CertificateCreateRequestJson;
+    } else if (firstParam instanceof CertificateCreateRequest) {
+      // Copy constructor
+      this._json = (firstParam as CertificateCreateRequest).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as CertificateCreateRequestJson;
+    }
+  }
+
+  /**
+   * Get trust object.
+   * @returns {TrustObjectRequestJson}
+   */
+  get trustObject(): TrustObjectRequestJson {
```

Comment:
this should return the wrapper object vs. JSON representation

Reply:
Fixed